frontend/public/src/components/forms/ProfileFormFields.js

```diff
@@ -222,12 +234,73 @@ export const LegalAddressFields = ({
           ))
           : null}
       </Field>
+
       <ErrorMessage
         name="legal_address.country"
         id="legal_address.country_error"
         component={FormError}
       />
     </div>
+    {findStates(values.legal_address.country, countries) ? (
+      <div className="form-group">
+        <label htmlFor="legal_address.state" className="font-weight-bold">
+          State<span className="required">*</span>
+        </label>
+        <Field
+          component="select"
+          name="legal_address.state"
+          id="legal_address.state"
+          className="form-control"
+          autoComplete="state"
+          aria-describedby="legal_address.state_error"
+        >
+          <option value="">-----</option>
+          {findStates(values.legal_address.country, countries)
+            ? findStates(values.legal_address.country, countries).map(
+              (state, i) => (
+                <option key={i} value={state.code}>
+                  {state.name}
+                </option>
+              )
+            )
+            : null}
+        </Field>
+        <ErrorMessage
+          name="legal_address.state"
+          id="legal_address.state_error"
+          component={FormError}
+        />
+      </div>
+    ) : null}
+    {isNewAccount ? (
+      <div className="form-group">
+        <label
+          htmlFor="user_profile.year_of_birth"
+          className="font-weight-bold"
+        >
+          Year of Birth<span className="required">*</span>
+        </label>
+        <Field
+          component="select"
+          name="user_profile.year_of_birth"
+          id="user_profile.year_of_birth"
+          className="form-control"
+          aria-describedby="user_profile.year_of_birth_error"
+        >
+          <option value="f">-----</option>
+          {reverse(range(seedYear - 120, seedYear - 14)).map((year, i) => (
```

Comment:
Currently, on RC, this renders the range 1903 - 2008. 

<img width="650" alt="image" src="https://user-images.githubusercontent.com/430126/220410992-e66c7a0c-0b6d-432a-bac0-2351a4a10673.png">

I think 1903 - 2009 would be more correct, since someone born 12/31/2009 would be over 13 years old today. 

I don't know how the range function handles boundary conditions, but maybe this should be:

```suggestion
          {reverse(range(seedYear - 120, seedYear - 13)).map((year, i) => (
```